MediaBrowser.Api/ItemRefreshService.cs

```diff
@@ -1,83 +0,0 @@
-using MediaBrowser.Controller.Configuration;
-using MediaBrowser.Controller.Library;
-using MediaBrowser.Controller.Net;
-using MediaBrowser.Controller.Providers;
-using MediaBrowser.Model.IO;
-using MediaBrowser.Model.Services;
-using Microsoft.Extensions.Logging;
-
-namespace MediaBrowser.Api
-{
-    public class BaseRefreshRequest : IReturnVoid
-    {
-        [ApiMember(Name = "MetadataRefreshMode", Description = "Specifies the metadata refresh mode", IsRequired = false, DataType = "boolean", ParameterType = "query", Verb = "POST")]
-        public MetadataRefreshMode MetadataRefreshMode { get; set; }
-
-        [ApiMember(Name = "ImageRefreshMode", Description = "Specifies the image refresh mode", IsRequired = false, DataType = "boolean", ParameterType = "query", Verb = "POST")]
-        public MetadataRefreshMode ImageRefreshMode { get; set; }
-
-        [ApiMember(Name = "ReplaceAllMetadata", Description = "Determines if metadata should be replaced. Only applicable if mode is FullRefresh", IsRequired = false, DataType = "boolean", ParameterType = "query", Verb = "POST")]
-        public bool ReplaceAllMetadata { get; set; }
-
-        [ApiMember(Name = "ReplaceAllImages", Description = "Determines if images should be replaced. Only applicable if mode is FullRefresh", IsRequired = false, DataType = "boolean", ParameterType = "query", Verb = "POST")]
-        public bool ReplaceAllImages { get; set; }
-    }
-
-    [Route("/Items/{Id}/Refresh", "POST", Summary = "Refreshes metadata for an item")]
-    public class RefreshItem : BaseRefreshRequest
-    {
-        [ApiMember(Name = "Recursive", Description = "Indicates if the refresh should occur recursively.", IsRequired = false, DataType = "bool", ParameterType = "query", Verb = "POST")]
-        public bool Recursive { get; set; }
```

Comment:
I don't think we should drop fields from API, even if they're unused... let's keep the compatibility.

Reply:
The `DirectoryService` doesn't allow nullable values so I set the default values